data_steward/spec/main.py

```diff
@@ -102,7 +102,11 @@ def get_full_result_log():
         hpo_id = hpo['hpo_id']
         hpo_bucket = gcs_utils.get_hpo_bucket(hpo_id)
 
-        obj_metadata = gcs_utils.get_metadata(hpo_bucket, RESULT_CSV)
+        try:
+            obj_metadata = gcs_utils.get_metadata(hpo_bucket, RESULT_CSV)
+        except:
```

Comment:
It might be better to catch the specific exception raised when the bucket doesn't exist and wrap the remainder of this code block. Don't worry about it, I may look into it on merge.